src/app/news/news-list/news-list.component.scss

```diff
@@ -0,0 +1,98 @@
+@media screen and (max-width: 575px) {
+   .list-news {
+      display: flex;
+      flex-direction: column;
+      list-style: none;
+      gap: 20px;
+      padding: 0;
+   }
+   .card-news {
+      max-width: 575px;
+   }
+   .wrapper-news {
+      display: flex;
+      flex-direction: column;
+      padding: 16px;
+      gap: 16px;
+   }
+   .img-news {
+      width: 100%;
+      height: 100%;
+   }
+}
+
+@media screen and (min-width: 576px) {
```

Comment:
please use scss variables from bootrstap (they're in `node_modules/bootstrap/scss/_variables.scss`). For example this `576px` can be replaced with `$grid-breakpoints` `sm`